### ISSUE TITLE
Spacer in Hardox

### ISSUE THREAD

**Issue by urpylka** [ONEARMY/precious-plastic-kit#29]
Please check location of the `SHRPRO-V1 1120.14-A Spacer 4mm`. Now it is in folder [Hardox_4mm](https://github.com/ONEARMY/precious-plastic-kit/tree/master/3.%20Build/5.%20Shredder%20Pro/Shredder%20Pro%20Files/2.%20Laser%20cut%20files/DXF/Hardox_4mm). But it will be placed in Steel 4mm folder (check [the blueprint](https://github.com/ONEARMY/precious-plastic-kit/blob/master/3.%20Build/5.%20Shredder%20Pro/Shredder%20Pro%20Files/3.%202D%20blueprints/SHRPRO-V1%201120.14-A%20Spacer%204mm.pdf)).

**Comment by darigovresearch** [comment]
Can any of the maintainers advise on this?

Not sure which one is correct.

@urpylka could you provide more context to what the issue is to help the maintainers resolve the issue?

Is it just that the spacer file should be moved from one folder to the other?

If so this may be a good first issue to make a pull request, that way all the maintainers need to do is to click one button to make the correction.

**Comment by urpylka** [comment]
Yes, I suggest to move file to another folder. I opened this issue to get feedback about that from maintainers. I've got answer in Discord (see image).

I can make PR if it will be accepted.

![photo_2021-05-31 21 52 26](https://user-images.githubusercontent.com/5459558/120227846-08c6f080-c25b-11eb-8d6c-56f7e523bad1.jpeg)

**Comment by darigovresearch** [comment]
Thanks for the clarification, you're welcome to put in a pull request but as we don't have privileges on the repository can't guarantee that it will be merged.

However there is only one way to find out if it will be merged and that is making a pull request

**Comment by darigovresearch** [comment]
Merging pull request #30 should resolve this issue, thanks for making the pull request
